crates/astria-conductor/src/tendermint.rs

```diff
@@ -99,3 +101,43 @@ impl TendermintClient {
             .wrap_err("failed reading server response as json")
     }
 }
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub struct UintString(pub u64);
+
+impl<'de> Deserialize<'de> for UintString {
+    fn deserialize<D: Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error> {
+        Ok(UintString(
+            String::deserialize(deserializer)?
+                .parse::<i64>()
+                .map_err(|e| D::Error::custom(format!("{e}")))?
+                .try_into()
+                .map_err(|e| D::Error::custom(format!("{e}")))?,
+        ))
+    }
+}
+
+impl Serialize for UintString {
```

Comment:
Do you need `serialize` given that you only derive `Deserialize` for the struct this is contained in?

Reply:
removed this type